RedisCore.Tests/RedisClientTestsBase.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace RedisCore.Tests
+{
+    public class RedisClientTestsBase
+    {
+        private static bool IsVstsBuild => Environment.GetEnvironmentVariable("TF_BUILD") != null;
+
+        protected static bool HasLocalRedis => !IsVstsBuild || Environment.GetEnvironmentVariable("LOCAL_REDIS") == "true";
+
+        private static readonly int[] BufferSizes = {16, 256, 65536};
+
+        private static IEnumerable<RedisClientConfig>  LocalTestConfigs()
+        {
+            foreach (var bufferSize in BufferSizes)
+            {
+                yield return new RedisClientConfig("127.0.0.1") {BufferSize = bufferSize};
+                //yield return new RedisClientConfig("192.168.0.64");
```

Comment:
Comment left in code

Reply:
That's my local redis on Linux VM

Reply:
Should it go into an environment variable, then defaulting to 127.0.0.1 if it does not present? That could serve as local configuration for your case.

Reply:
Good idea. Did that